gems/workflow/app/models/workflow/definition/process.rb

```diff
@@ -1,6 +1,8 @@
 module Workflow
   class Definition::Process < ApplicationRecord
+    has_many :instances, :class_name => 'Workflow::Instance::Process', :foreign_key => 'workflow_definition_process_id'
     has_many :steps
+    has_many :selected_processes
```

Comment:
this line seems wrong.

Reply:
Doesn't a process have many workflows?

Reply:
ah, yeah the line says `selected_processes`, you could put the `selected_workflows` association.  prob just a typo

Reply:
Think the join table you created is called `selected_processes` https://github.com/keithtom/wildflower-platform/blob/main/gems/workflow/db/migrate/20220806035158_create_workflow_definition_selected_processes.rb. Do you want me to rename to `selected_workflows`? I'm following this guide https://guides.rubyonrails.org/association_basics.html#the-has-many-through-association

Reply:
oh right, sorry.  that's a bad name... i wish I just put workflow_processes or something.  whatever.  let's just keep it as is for now.  but that name is confusing from the reverse direction...